pywemo/ouimeaux_device/insight.py

```diff
@@ -80,6 +81,11 @@ def today_kwh(self):
     def current_power(self):
         """Return the current power usage in mW."""
         return self.insight_params['currentpower']
+   
```

Comment:
It looks like there are still 3 blank spaces on line 84 of this file. Can you please make sure that line 84 is just a blank line?